docker-compose.yml

```diff
@@ -33,31 +26,17 @@ services:
     volumes:
       - ./frontend:/app
       - /app/node_modules
-    environment:
-      - CHOKIDAR_USEPOLLING=true
-
-  nginx:
```

Comment:
development env no need to use nginx, just use http://localhost:3000 to access to frontend and http://localhost:8000 to access to backend